static/sass/styles.scss

```diff
@@ -1789,6 +1789,15 @@ ol.p-stepped-list.no-full-stop
   text-align: left;
 }
 
+// Zoom effect for images on hover, used on /summit page
+.image-hover__zoom {
```

Comment:
We should try to follow BEM methodology for our class names, I would suggest something like `p-image-container__image--zoom`. Or if it is going to be used generically, `u-zoom-on-hover`